docs/config/ping-federate-example.md

```diff
@@ -9,23 +9,29 @@ Ensure you have installed the Identity Service before starting. You will also ne
 ## Configuration
 There are three main steps involved in configuring a PingFederate instance with the Identity Service:
 
-1. Obtain your parameters from PingFederate.
-2. Configure the Identity Service with your PingFederate parameters.
-3. Configure your PingFederate connection.
+1. Get the configuration values. 
+2. Configure your PingFederate connection.
+3. Configure the Identity Service with your PingFederate parameters.
 
-### 1. Obtain parameters from PingFederate
-1. Sign in to your PingFederate instance as a user with administrative privileges.
-2. Select **Server Settings** from the **System Settings** menu.
-3. Make a note of the value for *My Base URL* under the **Federation Info** heading.
-4. Click **Main** and select **Federation Settings** or **IdP Configuration**, then click **Protocol Endpoints**.
-5. Make a note of the *Redirect* values for *Single Logout (SLO) Service* and *Single Sign-on (SSO) Service* under the **SAML v2.0 Endpoints** heading.
-6. Combine the value of *My Base URL* (from Step 3) with *Single Logout (SLO) Service* and *My Base URL* (from Step 3) with *Single Sign-on (SSO) Service* ready to enter into the Identity Service.
+### Get the configuration values. 
```

Comment:
What the user is doing here isn't really getting "configuration values" (maybe in a very loose sense) but obtaining the Identity Service's cryptographic certificate.